app/res/values/strings.xml

```diff
@@ -12,12 +13,29 @@
     <!-- string name="ota_restore_url">https://pact.dimagi.com/provider/caselist</string -->
     <string name="key_server">https://pact.dimagi.com/keys/getkey</string>
     <string name="your_comment">Your comment</string>
-
+    
+    <string name="install_app">Install Another App</string>
+    <string name="uninstall_app">Uninstall</string>
+    <string name="archive_app">Archive</string>
+    <string name="update_app">Update</string>
+    <string name="verify_resources">Validate Multimedia</string>
+    <string name="select_app_prompt">Select an app to install</string>
+    <string name="resources_warning">Warning: Multimedia resources for this app have not been validated</string>
+    <string name="skipped_verification_warning">You did not validate multimedia resources for the app that was just
+         installed! This app will not be visible to users on the log-in screen until multimedia is validated.</string>
+    <string name="skipped_verification_warning_2">You did not validate multimedia resources for this app! The app will 
+        not be visible to users on the log-in screen until multimedia is validated.</string>
+    <string name="manager_instructions">Select an app to view its management options, or install a new app.</string>
+    <string name="old_profile_warning">WARNING: This app was installed from an old version of CommCare. For full
+        availability of app manager features (such as correct app name), please uninstall this app and reinstall 
+        with CommCare version 2.20 or later.</string>
+    <string name="uninstall_reboot_warning">CommCare will uninstall your app and then reboot to save changes.</string>

```

Comment:
No actually, I'm specifically warning them because I do a "reboot" of CommCare after uninstalling the app. It's a janky fix for potential inconsistencies that arise between the AppManagerActivity and the regular CommCare activity after uninstalling an app, but Clayton and I agreed that it was an ok solution for now.